### DISCUSSION TITLE
2024-01-10 Data Architecture Meeting

### DISCUSSION THREAD

**Discussion by jkbhagatio** [SainsburyWellcomeCentre/aeon_mecha#304] [Meeting Agendas]
- Overview of where things stand and todos before next Social0.1 cohorts start on both arenas (next week)
  - https://github.com/SainsburyWellcomeCentre/aeon_experiments/issues/406
  - https://github.com/SainsburyWellcomeCentre/aeon_experiments/issues/391
  - Daily Plots
- Social experiment ingestion
- Questions / comments about https://github.com/SainsburyWellcomeCentre/aeon_mecha/blob/social_schema/docs/examples/understanding_aeon_data_architecture.ipynb
- Updates on using Sleap1.3.3 models in Bonsai: https://swc-neuro.slack.com/archives/C03HPPP1LD9/p1704833417719159
- https://github.com/SainsburyWellcomeCentre/aeon_mecha/issues/290
  - What is the usecase of the DigitalBitmask reader? How is it different to BitmaskEvent?
- aeon_docs updates
- archiving exp02 database?

**Comment by ttngu207** [comment]
Decisions on the social01 schema
- move the `Pose` class from `schema.social.py` into `io.reader`
- the `social.py`  module will be used to define binder functions, devices etc. for the social experiments (things from `social01_schema.py`)
- schema will have the Devices explicitly specified in `schemas.py` for readability (instead of the list comprehension)
